rosdiagnostic/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rosdiagnostic)
+
+find_package(catkin REQUIRED COMPONENTS)
+
+catkin_package()
+
+catkin_python_setup()
+
+install(PROGRAMS
+  scripts/rosdiagnostic
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
```

Comment:
Also, you should test this locally (`catkin_make install; find install`), but I don't think you need to separately install the script— that's already covered by the `scripts` arg in `setup.py`.